en/cookbook/custom_field.rst

```diff
@@ -0,0 +1,187 @@
+                    expanded: # Add an expanded option with a default value: false (= rendered as a select) 
+                        default_value: false
+                    required: # Add a required option with a default value: true (The time zone will be required in the content form)
+                        default_value: true
+
+Now when an administrator includes a time zone field in a content type, he has the opportunity to decide if in
+this context the selection is single or multiple, if the time zone should be rendered as buttons or a select
+and whether it is mandatory or not. By default, the time zone is set to allow only a single selection in a
+select box. The selection of a zone is also mandatory. But the administrator can change this behavior by
+altering the values ​​of the options in the content form.
+
+.. image:: ../../images/timezone_options.png
+
+
+Configuring the default values
+------------------------------
+
+
+To further increase the possibilities of contribution, it may be interesting to give the administrator the
+ability to provide a default time zone in this field depending on the context of the content type. For example,
+perhaps in the case of a British platform, the time zone should be preset to "Europe/London". As always, to
+allow the administrator to enter a default value, we should describe which Symfony form type is to use to
+contribute it.
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    open_orchestra_backoffice:
+        field_types:
+            timezone_field:
+                label: open_orchestra.form.field_type.timezone
+                type: timezone
+                default_value:
+                    type: timezone # The default value is set via a timezone form type
+                    options: # The default value form type require some options
+                        label: open_orchestra_backoffice.form.field_type.default_value
+                        required: false # The administrator does not have to set default value if he don't want one
+                options:
+                    multiple:
+                        default_value: false
+                    expanded:
+                        default_value: false
+                    required:
+                        default_value: true
+
+The addition of the ``default_value`` entry includes a new field to configure the time zone. This default value
+is rendered as a timezone form type allowing the administrator to choose a time zone used as a default value
+in the content form. The default value field will receive the options ``label`` and ``required``. On the same
+way, other valid form type options may be included.
+
+
+You should now be able to see the new options:
+
+.. image:: ../../images/timezone_default_value.png
+
+Editing a content using a Timezone field
+----------------------------------------
+
+Finally when a contributor edit a content including a time zone he see it the way the adminsitrator configured
```

Comment:
little spelling pb :
When an administrator **creates** a new content type
This cookbook shows how **doing** this with
This **functionality** is also described in the configuration.
a new entry is added to the list of available field **types**.
As a field option allows an administrator to choose between several available **models**,
But in our example, it would lead to a **nonsense** to have the checkboxes required.
The default value field will receive the option **labels** and required
Finally**,** when a contributor **edits** a content**,** including a time zone**,** he see it the way the **administrator** configured it: